integrations/tensorflow_v1/classification.py

```diff
@@ -546,8 +546,8 @@ def parse_args():
             par.add_argument(
                 "--onnx-opset",
                 type=int,
-                default=11,
-                help="The onnx opset to use for export. Default is 11",
+                default=13,
```

Comment:
let's keep this as is for now unless tested

Reply:
Will do